### ISSUE TITLE
NTR: beta-alanine present in CSF (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8490]
New term request beta-alanine present in CSF (segal_030430155146): Cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000277
- label: Elevated CSF beta-alanine concentration
- parents: [Abnormal CSF amino acid concentration (HP:0500184)](https://hpo.jax.org/app/browse/term/HP:0500184)
- pmid: 
- def: The concentration of beta-alanine present in the cerebrospinal fluid (CSF) is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8490
- synonyms:
